lib/project_and_version_value.rb

```diff
@@ -74,7 +75,7 @@ def version_costs(proj_id, version_id)
   # @return [Issue] issue object
   def incomplete_project_issues(proj, basis_date)
     Issue.cross_project_scope(proj, 'descendants')
-      .where('start_date IS NOT NULL AND start_date <= ? AND due_date IS NOT NULL AND (closed_on IS NULL OR closed_on > ?)', basis_date, basis_date.to_time.end_of_day)
+      .where('start_date IS NOT NULL AND start_date <= ? AND due_date IS NOT NULL AND (closed_on IS NULL OR closed_on > ?)', basis_date, basis_date.end_of_day)
```

Comment:
__[rubocop]__
* Line is too long. [159/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)